backend/index.js

```diff
@@ -31,7 +26,9 @@ app.use(express.json())
 app.get('/', async (req, res) => {
   let collection = db.collection("posts");
   let results = await collection.find({})
-    .limit(50)
+    // .limit(50)
```

Comment:
hmm, weird, I'm not getting an error when I try the following

let results = await collection.find({}).limit(50)

<img width="363" alt="Screenshot 2023-12-04 at 3 01 22 PM" src="https://github.com/bhwong11/exploding-kittens/assets/5798861/835de735-df7e-4bb4-b32c-80a5a31023b7">